blocks/library/freeform/old-editor.js

```diff
@@ -159,7 +165,16 @@ export default class OldEditor extends Component {
 	render() {
 		const { id } = this.props;
 
+		const inspectorControls = (
+			<InspectorControls key="inspector">
+				<BlockDescription>
+					<p>{ __( 'The Class Editor block allows you add content using a WYSIWYG editor.' ) }</p>
```

Comment:
Typo: "Class" -> "Classic"

Reply:
Is "WYSIWYG" a widely understood acronym?

Reply:
The block title is `Classic Text`.
Maybe we should describe is like Core: `The Classic Text block gives you an editor that is similar to a word processor.`
https://github.com/WordPress/wordpress-develop/blob/4.8/src/wp-admin/edit-form-advanced.php#L366